packages/wrangler/src/pages/dev.ts

```diff
@@ -913,3 +811,185 @@ async function spawnProxyProcess({
 
 	return port;
 }
+
+/**
+ *
+ */
+function resolvePagesDevServerSettings(
```

Comment:
in need of a better naming here 🤔 I *think*. Maybe `reconcilePagesDevServerSettings` or `mergePagesDevServerSettings`? Or is `resolvePagesDevServerSettings` descriptive enough?